modules/virt-about-fusion-access-san.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * virt/fusion_access_SAN/about-fusion-access-san.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="about-fusion-access-san_{context}"]
+= About {IBMFusionFirst}
+
+{IBMFusionFirst} is a solution that provides a scalable clustered file system for enterprise storage, primarily designed to offer access to consolidated, block-level data storage. It presents storage devices, such as disk arrays, to the operating system as if they were direct-attached storage.
+
+This solution is particularly geared towards enterprise storage for Red Hat {VirtProductName} and leverages existing Storage Area Network (SAN) infrastructure. A SAN is a dedicated network of storage devices that is typically not accessible through the local area network (LAN).
+
+[discrete]
```

Comment:
Discrete headings are not supported in DITA. Since this is a concept module, this will need to become a full section. Looking at the contents of the section, I don't think it is a problem if it makes it in the table of contents.
```suggestion
```